connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/ReflectionScanner.java

```diff
@@ -1,149 +1,149 @@
-/*
```

Comment:
It was happening when I was rolling back changes via IDE. I think it somehow adding carriage returns to the lines since its on windows. Does not normally happen If I do operation with WSL.

Reply:
Ah yeah the line endings could cause a big diff. Thanks for looking into it!